src/components/Add_Ride.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useEffect } from 'react';
+import Select from 'react-select';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './component.css';
+import AddLocation from './Add_Location';
+import {Button, InputGroup, Label, Input} from 'reactstrap';
+
```

Comment:
Please make library imports at top and remove blank lines,

Reply:
I have Done The changes, Please Review It.